CMakeLists.txt

```diff
@@ -89,7 +89,13 @@ set(LLVM_EXTERNAL_LIT ${LLVM_TOOLS_BINARY_DIR}/llvm-lit CACHE STRING "Command us
 
 if(LINK_WITH_FIR)
```

Comment:
Why are we still supporting "LINK_WITH_FIR"?

Reply:
I feel it is safer to keep it until f18 lands in LLVM and people are used to building with LLVM (it's On by default). I am OK removing it, I will send a mailing list email to tell people they have to update LLVM builds when merging this. I will also ask if anyone is using `LINK_WITH_FIR=Off` and remove that in a later patch if nobody is attached to it.

Reply:
I think it would be good for everyone to figure out how to build before the move to the LLVM repo, so I agree with removing this.

Reply:
I think you should just set a date and remove it.